Classes/Services/Transfer/DocumentTransferManager.php

```diff
@@ -288,23 +295,23 @@ public function retrieve($remoteId)
     public function delete($document, $state)
     {
 
-        $document->setTransferStatus(Document::TRANSFER_QUEUED);
-        $this->documentRepository->update($document);
+        //$document->setTransferStatus(Document::TRANSFER_QUEUED);
+        //$this->documentRepository->update($document);
 
             switch ($state) {
```

Comment:
Case statements are essentially returns. Switch statement can be replaced by multiple if-then.

Reply:
The switch statement has been replaced.